### PR TITLE
refactor: de-duplicate event definitions

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: interbtc-spec/docs/source/spec/vault-registry.rst [hunk 1]
```diff
@@ -638,11 +638,7 @@ Specification
 
 *Returns*
 
-* A tupple of the new total ``toBeReplacedTokens`` and ``replaceCollateral``.
-  
-*Events*
```
Comment (sander2) [inline interbtc-spec/docs/source/spec/vault-registry.rst:643]: Why did you remove this event?

Comment (gregdhill) [reply interbtc-spec/docs/source/spec/vault-registry.rst:643]: Wasn't specified below, added in a separate commit. Also included a missing event here and in the implementation, see: https://github.com/interlay/interbtc/pull/191

**Review by sander2 — APPROVED** [review]
